plugins/profiles.koplugin/main.lua

```diff
@@ -69,7 +76,12 @@ function Profiles:getSubMenuItems()
                             text = _("Save"),
                             callback = function()
                                 local name = name_input:getInputText()
-                                if not self:newProfile(name) then
+                                if name == autostart_profile then
+                                    UIManager:show(InfoMessage:new{
+                                        text =  T(_("Reserved name can not be used: %1"), autostart_profile),
```

Comment:
Is there a need to overload the profiles object with the autostart info?

You can instead mage a new global setting `autostart_profiles` with `G_reader_settings:` and use the following to add/remove items from it:
https://github.com/koreader/koreader/blob/bc133e70d3e701907e9a985deeb34816ca65e878/frontend/luasettings.lua#L228-L241